### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by dpshelio**

### CODE REVIEW

**Comment by dpshelio** [comment]
Note that the last commit has TODOs!!

File: ch00python/023types.ipynb [hunk 1]
```diff
@@ -1299,6 +1311,49 @@
    "source": [
     "zero, one, two, three = range(2)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "FIXME: put extra notes about unpacking in python3"
```
Comment (dpshelio) [inline ch00python/023types.ipynb]: :warning:

File: ch00python/023types.ipynb [hunk 2]
```diff
@@ -889,6 +862,45 @@
...
+   "metadata": {},
+   "source": [
+    "### Ranges"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "FIXME: Another type is range which gives you numbers as you need them."
```
[the remark this answers is not in the document]
Comment (ageorgou) [reply ch00python/023types.ipynb]: Done in c963dfd.

File: ch00python/023types.ipynb (hunk 1 above)
Comment (ageorgou) [reply ch00python/023types.ipynb]: Done in 4ceb91e.

File: ch00python/025containers.ipynb [hunk 3]
```diff
@@ -686,7 +686,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Try the [visualiser](http://www.pythontutor.com/visualize.html#code=x%3D%5B%5B'a','b'%5D,'c'%5D%0Ay%3Dx%0Az%3Dx%5B0%3A2%5D%0A%0Ax%5B0%5D%5B1%5D%3D'd'%0Az%5B1%5D%3D'e'&mode=display&origin=opt-frontend.js&cumulative=false&heapPrimitives=true&textReferences=false&py=2&rawInputLstJSON=%5B%5D&curInstr=0) again."
+    "Try the [visualiser](http://www.pythontutor.com/visualize.html#code=x%3D%5B%5B'a','b'%5D,'c'%5D%0Ay%3Dx%0Az%3Dx%5B0%3A2%5D%0A%0Ax%5B0%5D%5B1%5D%3D'd'%0Az%5B1%5D%3D'e'&mode=display&origin=opt-frontend.js&cumulative=false&heapPrimitives=true&textReferences=false&py=2&rawInputLstJSON=%5B%5D&curInstr=0) again.\n",
+    "FIXME: add something about `copy.deepcopy`"
```
[the remark this answers is not in the document]
Comment (ageorgou) [reply ch00python/025containers.ipynb]: Done in 076f62e?

**Comment by ageorgou** [comment]
Looks good @dpshelio, and I think I've covered the FIXMEs.  I will check the images along with other issues for these chapters in another PR.

**Comment by ageorgou** [comment]
While the build succeds, Travis has the following eror on the last step:
```
$ git push -f -u origin gh-pages
ERROR: We're doing an SSH key audit.
Reason: unverified automatically (private key found in a public repository)
Please visit https://github.com/settings/keys/17645879
to approve this key so we know it's safe.
Fingerprint: 40:2c:72:43:84:69:73:6d:a9:e0:f3:b4:aa:d6:b3:0d
fatal: Could not read from remote repository.
Please make sure you have the correct access rights
and the repository exists.
```

This won't be an issue after #113, but maybe we should check it anyway...

**Pull request merged**